ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -83,7 +70,7 @@ public class Worker extends RemoteCompactorThread implements MetaStoreThread {
   static private long SLEEP_TIME;
 
   private String workerName;
-  private final CompactorFactory compactorFactory;
+  protected final CompactorFactory compactorFactory;
```

Comment:
is that use at all?

Reply:
Not used in Worker. Moved to CompactionExecutor.